src/App.tsx

```diff
@@ -1,14 +1,56 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
+  const [todoId, setTodoId] = useState(0);
+  const [userId, setUserId] = useState<number>(0);
+  const [query, setQuery] = useState('');
+
+  const selectedTodo = todos
+    .find(todo => todo.id === todoId);
+
+  useEffect(() => {
+    getTodos()
+      .then(tds => {
+        setTodos(tds);
+      });
+  }, []);
+
+  const visibleMovies = (visibleTodos: Todo[], filt: string) => {
```

Comment:
use full variable names
```suggestion
  const visibleMovies = (visibleTodos: Todo[], filter: string) => {
```